libsolidity/formal/ModelCheckerSettings.h

```diff
@@ -160,15 +176,9 @@ struct ModelCheckerSettings
 	bool operator!=(ModelCheckerSettings const& _other) const noexcept { return !(*this == _other); }
 	bool operator==(ModelCheckerSettings const& _other) const noexcept
 	{
-		return
-			contracts == _other.contracts &&
-			divModNoSlacks == _other.divModNoSlacks &&
-			engine == _other.engine &&
-			invariants == _other.invariants &&
-			showUnproved == _other.showUnproved &&
-			solvers == _other.solvers &&
-			targets == _other.targets &&
-			timeout == _other.timeout;
+		return contracts == _other.contracts && divModNoSlacks == _other.divModNoSlacks && engine == _other.engine
+			   && invariants == _other.invariants && showUnproved == _other.showUnproved && solvers == _other.solvers
+			   && targets == _other.targets && timeout == _other.timeout;
```

Comment:
This PR has a lot of unnecessary reformatting that just makes it harder to see actual changes. Can you please remove formatting only-changes and make sure that the code you're adding matches the style of the code around it?

Reply:
@cameel, removed extra reformatting and fixed code style. I had some troubles with formatting in my ide. That's why there were a lots of unnecessary reformatting. But, now it seems to be fixed.